src/misc/lv_style.h

```diff
@@ -155,120 +154,110 @@ typedef union {
  * Enumeration of all built in style properties
  */
 typedef enum {
-    LV_STYLE_PROP_INV                = 0,
+    LV_STYLE_PROP_INV               ,
 
     /*Group 0*/
-    LV_STYLE_WIDTH                   = 1 | LV_STYLE_PROP_LAYOUT_REFR,
```

Comment:
At least the each groups' first index should be kept to form groups. 

The idea is to group similar properties (e.g. `text_...`) so [get_prop_core()](https://github.com/lvgl/lvgl/blob/master/src/core/lv_obj_style.c#L555) can skip styles that doesn't have properties related to the one being searched.

Reply:
True. I also put the indexes back so that it's easy to see when the group overflows.